apps/discovery_api/lib/discovery_api/event/event_handler.ex

```diff
@@ -124,6 +126,64 @@ defmodule DiscoveryApi.Event.EventHandler do
     end
   end
 
+  def handle_event(%Brook.Event{type: dataset_access_group_associate(), author: author, data: %DatasetAccessGroupRelation{} = relation}) do
+    Logger.warn(fn ->
+      "Handling dataset-access-group association: `Dataset: #{relation.dataset_id} Access Group: #{relation.access_group_id}`"
+    end)
+
+    dataset_access_group_associate()
+    |> add_event_count(author, relation.dataset_id)
+
+    with {:ok, dataset} <- Brook.get(@instance_name, :models, relation.dataset_id),
+         model <- Mapper.add_access_group(dataset, relation.access_group_id) do
+      Elasticsearch.Document.update(model)
+
+      Logger.warn(fn ->
+        "Successfully handled dataset-access-group association message: `Dataset: #{relation.dataset_id} Access Group: #{
+          relation.access_group_id
+        }`"
+      end)
+
+      merge(:models, model.id, model)
+      clear_caches()
+
+      :discard
+    else
+      {:error, reason} ->
+        Logger.error("Unable to process message `#{inspect(relation)}` from `#{inspect(author)}` : ERROR: #{inspect(reason)}")
+        :discard
+    end
+  end
+
+  def handle_event(%Brook.Event{type: dataset_access_group_disassociate(), author: author, data: %DatasetAccessGroupRelation{} = relation}) do
+    Logger.debug(fn ->
+      "Handling dataset-access-group disassociation: `Dataset: #{relation.dataset_id} Access Group: #{relation.access_group_id}`"
+    end)
+
+    dataset_access_group_disassociate()
+    |> add_event_count(author, relation.dataset_id)
+
+    with {:ok, dataset} <- Brook.get(@instance_name, :models, relation.dataset_id),
+         model <- Mapper.remove_access_group(dataset, relation.access_group_id) do
+      Elasticsearch.Document.update(model)
+
+      Logger.debug(fn ->
```

Comment:
the associate event uses logger.warn, could we unify the two so that the both use `.debug` or both use `.warn`

don't want a false indication that either didn't happen if looking while log level is default

Reply:
good point! Set to all the logs to debug to follow the established pattern!